### PR TITLE
Templates: add new templates from batch 1

### LINKED ISSUE #7234: Templates Project: Batch 1 (1-10)
## Task Description

[**Figma link**](https://www.figma.com/file/OQIT5ISzdQVvRD7Aor8ajA/Web-Stories-50-Templates?node-id=1682%3A0)

Screenshot 2021-04-16 at 17 33 02

This batch includes 10 new templates:

1. Entertainment
2. Entertainment
2. Travel
3. Travel
3. Technology
4. Technology
5. Fashion & Beauty
6. Arts & Craft
7. Home & Garden
8. Health & Wellness

### PR DESCRIPTION
**Pull request by divyarajmasani**
## Summary

Adds 9 new story templates.

- Almodo’s Films
- Pizzas in NYC
- 12 Hours in Barcelona
- Ultimate Comparison
- Fitness Apps Ranked
- Street Style On The Go
- Plant Based Dyes DIY
- Indoor Garden Oasis DIY
- Belly Fat Workout

Meta data for the templates were provided along with the designs in the Figma file itself, this includes sticker names, title, tags, description and colour palette.

## Relevant Technical Choices

- Adds stickers and story template JSON
- Uses title/alt texts for videos and images from the original source i.e. getty / unsplash / pexels.
- Updates `colorList` component, shown on the template details modal, to include new lighter shades to add border around.

## Testing Instructions

New templates should be available under 'Explore Templates' page.
`/wp-admin/edit.php?post_type=web-story&page=stories-dashboard#/templates-gallery`

Screenshot 2021-07-09 at 6 31 04 PM

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #7234
Companion PR: #8265

### CODE REVIEW

**Comment by swissspidy** [comment]
@googlebot I consent

**Comment by swissspidy** [comment]
Wen through all the new templates and they look good to me so far. Just pushed some minor, slightly unrelated string changes, but that's it.

I'll make sure #7885 covers these new templates as well.

**Review by swissspidy — APPROVED** [review]

File: packages/templates/src/raw/indoor-garden-oasis/template.json [hunk 1]
```diff
@@ -0,0 +1,3898 @@
...
+          "rotationAngle": 0,
+          "lockAspectRatio": true,
+          "x": 214,
+          "y": 107,
+          "width": 147,
+          "height": 10,
+          "scale": 100,
+          "focalX": 50,
+          "focalY": 50,
+          "sticker": { "type": "homeGardenLine01" },
```
Comment (swissspidy) [inline packages/templates/src/raw/indoor-garden-oasis/template.json]: @divyarajmasani This sticker does not exist.

Comment (divyarajmasani) [reply packages/templates/src/raw/indoor-garden-oasis/template.json]: Updated the sticker name.

**Pull request merged**